appstore/appstore/settings/base.py

```diff
@@ -177,12 +177,25 @@
 
 DB_DIR = Path(os.environ.get("OAUTH_DB_DIR", DJANGO_PROJECT_ROOT_DIR))
 DB_FILE = Path(os.environ.get("OAUTH_DB_FILE", "DATABASE.sqlite3"))
-DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": DB_DIR / DB_FILE,
+
+if DEV_PHASE == "prod":
+    DATABASES = {
+        "default": {
+            "ENGINE": f"django.db.backends.{os.environ.get('PG_DB_ENGINE', 'postgresql')}",
+            "NAME": os.environ.get("PG_DB_DATABASE", "appstore-oauth"),
+            "USER": os.environ.get("PG_DB_USERNAME", "renci"),
+            "PASSWORD": os.environ.get("PG_DB_PASSWORD", "renci"),
+            "HOST": os.environ.get("PG_DB_HOST", "appstore-postgresql"),
+            "PORT": os.environ.get("PG_DB_PORT", "5432"),
+        }
+    }
+else:
```

Comment:
why do we not want to have a postgresql database for dev as well to be consistent?

Reply:
It can be made available for local development as well. In that case, all developers should be able to setup their own postgresql instances locally.
When appstore is deployed anywhere using the helm charts, the default database would always be postgresql.

Reply:
that works but can we not make postgresql creation part of helx deployment as in a helx chart?

Reply:
I guess we can include a docker-compose to setup postgresql locally, should be straightforward.

Reply:
Added a basic postgreql docker-compose and a make rule for installing it.